pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -335,6 +335,20 @@ type ScrapeConfigSpec struct {
 	// ProxyConfig allows customizing the proxy behaviour for this scrape config.
 	// +optional
 	v1.ProxyConfig `json:",inline"`
+	// Specifies the validation scheme for metric and label names.
+	// Enter "UTF8" for full UTF-8 support, or "Legacy" for letters, numbers, colons, and underscores.
+	// Blank value will be utf8.
+	//
+	// It requires Prometheus >= v3.0.0.
+	//
+	// +optional
+	MetricNameValidationScheme *v1.NameValidationSchemeOptions `json:"metricNameValidationScheme,omitempty"`
```

Comment:
Hi @nutmos maybe is better for consistency to keep the naming equal to whats in [prometheus_types.go](https://github.com/kubeservice-stack/prometheus-operator/blob/main/pkg/apis/monitoring/v1/prometheus_types.go), so remove the `metric` from `namevalidationscheme` and `nameescapingscheme`

Reply:
It's (almost) never 1:1 because Prometheus and Kubernetes have different case conventions (snake_case vs. CamelCase).